Multisig/UI/Settings/OwnerKeyManagement/RequestToAddOwner/IdenticonInfoView.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  KeyInfoView.swift
+//  Multisig
+//
+//  Created by Vitaly on 13.06.22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class IdenticonInfoView: UINibView {
+
+    @IBOutlet weak var blockie: IdenticonView!
```

Comment:
Private ?

Reply:
this is how xcode creates outlet connections by default. I agree that this is related to implementation details and should be hidden.